uvahelpr_web/marketplace/tests.py

```diff
@@ -1,3 +1,7 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.core.urlresolvers import reverse
+from uvahelpr_web.models import Order, User
 
 # Create your tests here.
+
+
```

Comment:
don't think you need this changes... I think it's safe to leave them here for now, but they might conflict with abhi's testing code